src/optimization/inlineConstructors.ml

```diff
@@ -736,17 +736,30 @@ let inline_constructors ctx original_e =
 				begin try
 					let is_user_variable iv = match iv.iv_var.v_kind with VUser _ | VInlined -> true | _ -> false in
 					let iv = List.find is_user_variable io.io_aliases in
-					(get_pretty_name iv) ^ "_" ^ fname;
+					(get_pretty_name iv) ^ "." ^ fname;
```

Comment:
I'm not super familiar with the context here but I don't think we want to have dots in our variable names.

Reply:
I replaced them in `v.v_name <- Str.global_replace (Str.regexp "\\.") "_" vname;`.
I'm having some difficulty when trying to get the variable path, maybe I should return a string list from get_pretty_name and concat it. Will do.